server/@types/adjustments/index.d.ts

```diff
@@ -290,6 +265,104 @@ export interface components {
       /** @description The NOMIS ID of the person this adjustment applies to */
       person: string
     }
+    /** @description The adjustment and its identifier */
+    AdjustmentDto: {
+      /**
+       * Format: uuid
+       * @description The ID of the adjustment
+       */
+      id?: string
+      /**
+       * Format: int64
+       * @description The NOMIS booking ID of the adjustment
+       */
+      bookingId: number
+      /** @description The NOMIS ID of the person this adjustment applies to */
+      person: string
+      /**
+       * @description The type of adjustment
+       * @enum {string}
+       */
+      adjustmentType:
+        | 'REMAND'
+        | 'TAGGED_BAIL'
+        | 'UNLAWFULLY_AT_LARGE'
+        | 'LAWFULLY_AT_LARGE'
+        | 'ADDITIONAL_DAYS_AWARDED'
+        | 'RESTORATION_OF_ADDITIONAL_DAYS_AWARDED'
+        | 'SPECIAL_REMISSION'
+        | 'UNUSED_DEDUCTIONS'
+      /** @description Human readable text for type of adjustment */
+      adjustmentTypeText?: string
+      /**
+       * Format: date
+       * @description The end date of the adjustment
+       */
+      toDate?: string
+      /**
+       * Format: date
+       * @description The start date of the adjustment
+       */
+      fromDate?: string
+      /**
+       * Format: int32
+       * @deprecated
+       * @description The number of adjustment days, Deprecated: Use daysTotal instead
+       */
+      days?: number
```

Comment:
Can we manually remove the deprecated fields here, just to test all usages have been removed?